README.md

```diff
@@ -354,8 +356,33 @@ nim_gc_heap_instance_occupied_bytes[type_name]
 The `process_*` metrics are only available on Linux, for now.
 
 `nim_gc_heap_instance_occupied_bytes` is only available when compiling with
-`-d:nimTypeNames` and holds the top 5 instance types, in reverse order of their
-total heap usage, at the time the metric is created.
+`-d:nimTypeNames` and holds the top 10 instance types, in reverse order of
+their total heap usage (from all threads), at the time the metric is created.
+Since this set changes with time, you'll see more than 10 types in Grafana.
+
+These system metrics are being updated automatically when a user-defined metric
```

Comment:
this remains a fundamentally problematic approach because it doesn't happen at regular intervals - it will produce biased and stale data that is hard to interpret, and hard to understand _why_ it's wrong. It works in some cases where the application updates frequently enough (though it breaks the pay-for-what-you-use principle), but not across all "normal" applications - what's worse is that it breaks in a way that's hard to understand - you'd have to simultaneously monitor the application with a _working_ metrics system to understand that the numbers are completely wrong.